### PR TITLE
adding example testnet configs

### PR DESCRIPTION
**Pull request by parithosh**

### CODE REVIEW

File: example-testnet/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Example testnet
+
+This directory contains the configs required to run an example testnet. 
+
+# Assumption
+- This setup expects that you already have deposits made and config data generated
+- This setup is ideally aimed at public testnets and helps quickly provision validators
+
+# Setup
+- Ensure this repo is cloned locally and that the submodules have been pulled
```
Comment (skylenet) [inline example-testnet/README.md:10]: There don't seem to be any submodules in this repo.

File: example-testnet/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Ensure ansible has been installed
+- Ensure SSH access to all servers
+
+# Usage
+- Modify the `example-testnet/inventory/dynamic.py` file with the correct tags and client distribution, if a dynamic inventory
```
Comment (skylenet) [inline example-testnet/README.md:15]: Maybe it would be nice to just use a simple static inventory for the example.

**Review by skylenet — COMMENTED** [review]
Let's keep this PR a bit longer open so that I can test this end to end hopefully next week.

**Review by protolambda — APPROVED** [review]
LGTM, let's ship a merge testnet

**Pull request merged**